recipe/meta.yaml

```diff
@@ -11,18 +11,22 @@ source:
   sha256: a777e07633d83d546c55706420179551c8e01075b53c497dcf8ae4036766bc66
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
     - setuptools
     - cython
-    - numpy  x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
   run:
     - python
-    - numpy  x.x
+    - numpy >=1.7  # [py27]
+    - numpy >=1.9  # [py35]
+    - numpy >=1.11  # [py36]
```

Comment:
But if there ever was a numpy 1.7 for python 3.5 or 3.6 the package built using this recipe would NOT work with them because numpy does not guarantee forward compatibility.  

These lines properly describe the numpy requirement for the various Python versions.  I don't think depending on the repository not having certain versions of a package is something we want to depend upon.